testing/src/main/graphql/betterbotz/schema.json

```diff
@@ -2088,8 +2088,7 @@
                 "kind": "SCALAR",
                 "name": "Int",
                 "ofType": null
-              },
-              "defaultValue": "-1"
```

Comment:
why this change?

Reply:
Good question!

There was a bug in `MutationOptions` where the default value for ttl property was `-1`. But then, `buildStatement()` implementations used `options.get("ttl") != null` condition to check whether ttl was present. So, when the options where defined but the TTL was not set, it would fail by setting TTL to -1.